com.unity.perception/Runtime/GroundTruth/Labelers/CameraLabeler.cs

```diff
@@ -60,12 +123,61 @@ internal void Init(PerceptionCamera newPerceptionCamera)
                 sensorHandle = newPerceptionCamera.SensorHandle;
                 Setup();
                 isInitialized = true;
+
+                if (supportsVisualization)
+                {
+                    InitPanel();
+                }
             }
             catch (Exception)
             {
                 this.enabled = false;
                 throw;
             }
         }
+
+        private GameObject GetCanvas()
+        {
+            var canvas = GameObject.Find("Canvas");
```

Comment:
I was doing this in case some user was using a scene that already had a canvas object as a part of it. Also, the use case that multiple visualizers are being used, we don't want to have multiple canvases being created (although that could be solved using a static canvas variable). I agree that we should avoid using GameObject.Find() because of performance reasons. But in this case it should only be called once during initialization.

Reply:
Added a cached member to the CameraLabeler class to make sure that the `GameObject.Find()` is only called at most once per visualizer at init time.

Reply:
It feels a bit dirty to reuse the users' canvas if they have one, especially since the user might just end up doing any number of things that we don't want to that container, like delete it. At best the user would end up seeing a bunch of extra objects attached to their canvas. I just tried it out, and having multiple canvases in your scene works fine. Is there a benefit to reuse the existing canvas?

Reply:
One more note on this. In my head, the implementation details of any development tool should be hidden from the user whenever possible. This can be a real challenge in Unity, where user options in the project can sometimes break the implementation of your feature. (for example, what happens if the user changes their canvas to World Space?) It's not always possible to avoid this, but it's something to be conscious of.

Reply:
Reimplemented a lot of the way I am allocating UI objects, how the canvas operates, etc. But I agree with you about having our own canvas no matter what.